app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -1200,6 +1222,17 @@ class HomeFragment : Fragment() {
         }
     }
 
+    // We want to show the animation in a time when the user less distracted
+    // The Heuristics are:
+    // 1) The animation hasn't shown before.
+    // 2) The user has onboarded.
+    // 3) It's the third time the user enters the app.
+    @Suppress("MagicNumber")
+    private fun shouldAnimateLogoForWallpaper() =
+        shouldEnableWallpaper() && context?.settings()?.shouldAnimateFirefoxLogo == true
+            && onboarding.userHasBeenOnboarded()
+            && context?.settings()?.numberOfAppLaunches == 3
```

Comment:
@MatthewTighe just to make you aware, I added some extra logic, after the product input. They want to show the animation when the user is less distracted with other parts of the app, specifically after onboarding has been finished.

Reply:
Thanks for the update! Looks good to me